csrc/ir/internal_nodes.h

```diff
@@ -2288,4 +2288,50 @@ class MatmulOp : public Expr {
       const std::vector<PolymorphicValue>& inputs) const override;
 };
 
+//! Linear Operator to be expression evaluated without decomposition.
```

Comment:
You could leave a comment here that points to the torch docs for semantics of this node: https://pytorch.org/docs/stable/generated/torch.nn.functional.linear.html#torch.nn.functional.linear